sklearn/model_selection/validate.py

```diff
@@ -0,0 +1,905 @@
+"""
```

Comment:
The ordering of functions in this file is very different from master (it made diffing a pain), I wonder whether there was any particular reason for it?